dev-lisp/ecls/ecls-20.4.24.ebuild

```diff
@@ -80,15 +84,12 @@ src_compile() {
 		touch build/TAGS
 	fi
 
-	#parallel make fails
-	emake -j1
+	emake
 }
 
 src_install() {
-	emake DESTDIR="${D}" install
+	default
 
-	dodoc README.md CHANGELOG
-	dodoc "${FILESDIR}"/README.Gentoo
```

Comment:
What about that `README.Gentoo` file? Does it no longer apply?

If it still does, the ebuild should do it properly and inherit `readme.gentoo-r1.eclass`.

Reply:
For the record, I removed that line about `README.gentoo` in `src_install` because I included it in the `DOCS` variable that is installed with `default` in EAPI 7.